lang/cs/Org.Apache.REEF.Tang/Implementations/ClassHierarchy/NamedParameterNodeImpl.cs

```diff
@@ -44,6 +47,15 @@ public class NamedParameterNodeImpl : AbstractNode, INamedParameterNode
             this.documentation = documentation;
             this.shortName = shortName;
             this.defaultInstanceAsStrings = defaultInstanceAsStrings;
+            this.alias = alias;
+            this.aliasLanguage = aliasLanguage;
+        }
+
+        public NamedParameterNodeImpl(INode parent, String simpleName,
```

Comment:
Alias can be a Tang feature. I don't think we should treat it differently from oters. However, if we don't need for clients to call class hierarchy and Node related methods, we should make them all as internal. Do a in a separate Jira.